list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The current implementation directly uses raw SQL queries without any error handling or data validation.

Recommendations:
- **Use Prepared Statements**: Utilize prepared statements to prevent SQL injection.
- **Refactor to Object-Oriented Design**: Implement a `BookService` class to handle book-related operations. This class can use a `Database` class for database interactions.